recipes/gmsh/tests.patch

```diff
@@ -0,0 +1,2 @@
+sed -i '1919i \ \ list(REMOVE_ITEM TESTFILES "${CMAKE_SOURCE_DIR}/tutorial/t16.geo")' CMakeLists.txt
```

Comment:
I meant a bit more like this:
https://github.com/ilastik/ilastik/wiki/Creating-patches-for-Conda-recipes

Reply:
My bad, thought it was a bash script preprocessed by the environment...

Reply:
Sorry it is a mistake in my diff, I have done it from a local copy and not with git diff...